sqlstore/src/test/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePlatformDaoTest.java

```diff
@@ -180,4 +181,11 @@ Platform makePlatform(String instrumentModel, String description, Integer numCon
     platform.setPlatformType(PlatformType.get("PacBio"));
     return platform;
   }
+
+  @Test
+  public void testListActivePlatformTypes() throws Exception {
```

Comment:
Probably worth including a negative test for SOLID or 454.

Reply:
Added